docs/configuration/translation-strings.md

```diff
@@ -0,0 +1,65 @@
+# Translation Strings
+
+> Translation Strings are multilingual key-value pairs that you can use throughout the app. They enable you to translate
+> things like dropdown options, placeholder text, field notes, and more.
+
+[[toc]]
+
+## How It Works
+
+<video autoplay muted loop controls>
+<source src="https://cdn.directus.io/docs/v9/configuration/translation-strings/translation-strings-20220615A/translation-strings-20220615A.mp4" type="video/mp4" />
+</video>
+
+When creating a Field, you may want to add a note or other description along with it to clarify its context or purpose.
+While you always have the option to type in a static value, Translation Strings let you group together translations
+which display dynamically, whenever the app language is changed. There are two locations where the language could be
```

Comment:
> There are two locations where the language

This is useful _secondary_ information, but over half of this intro is about where/how you set up or change languages. Those instructions should be in a different page/section (eg: "How to change app languages"). Then we can just say "Translation Strings allow for setting multilingual text/values that display based on the **App's configured language**". Where the bold text is a link to those other docs.

Reply:
Yes. I agree. But we need to add this onto Michelle's News release asap. And we don't have those other docs yet, and adding the videos means.

I also think this is important *(albeit secondary)* information to fully seeing what these translation strings do. I moved this info into the final paragraph under `## Use a Translation String`, and cut the sentence lengths in half by adding links.